pkgs/development/tools/misc/gnum4/default.nix

```
@@ -6,7 +6,8 @@
 # files.
 
 stdenv.mkDerivation {
-  name = "gnum4-1.4.18";
+  pname = "gnum4";
+  version = "1.4.18";
 
   src = fetchurl {
     url = "mirror://gnu/m4/m4-1.4.18.tar.bz2";
```

```suggestion
    url = "mirror://gnu/m4/m4-${version}.tar.bz2";
```
this is my preferred style to only substitute the version number.